.eslintignore

```diff
@@ -0,0 +1 @@
+Enums.ts
```

Comment:
this file is similar to `.gitignore`

decided to ignore the whole Enum file because there would have been a lot of suppress comments there lol